src/Pages/contactUs/components/contactUs.scss

```diff
@@ -0,0 +1,208 @@
+        position: relative;
+        text-decoration: none;
+        width: 100%;
+    }
+
+    #LinkToAboutUs li a, #LinkToMarketPlace li a, #LinkToHome li a {
+        display: block;
+        text-align: center;
+        padding: 16px;
+        text-decoration: none;
+    }
+
+    #LinkToAboutUs button, #LinkToMarketPlace button, #LinkToHome button {
+        background-color: #F0DDCE;
+        font-size: 20px;
+        border-radius: 6px;
+        border: none;
+        margin: 0px auto;
+        display: block;
+        width: 172px;
+        height: 100px;
+        position: relative;
+        box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+    }
+
+    #sendButton {
+        position: relative;
+        left: 10%;
+    }
+
+    #LinkToAboutUs button:hover, #LinkToMarketPlace button:hover, #LinkToHome button:hover {
+        background-color: #eeee;
+    }
+
+    button:hover {
+        background-color: #eeee;
+    }
+
+    textarea {
+        border: 2px solid #447399;
+        width: 75%;
+        height: 400px;
+        resize: none;  
+        display: block;
+        font-size: 22px;
+        margin: 0px auto;
+
+    }
+
+    input {
+        border: 2px solid #447399;
+        width: 25%;
+        height: 50px;
+        resize: none;  
+        display: block;
+        position: relative;
+        left: 12.5%;
+        font-size: 18px;
+        margin-bottom: 3%;
+    }
+
+    .sendButton {
+        background-color: #F0DDCE;
+        font-size: 20px;
+        border-radius: 6px;
+        border: none;
+        margin: 0px auto;
+        display: block;
+        width: 100px;
+        height: 50px;
+        position: relative;
+        box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+        margin-top: 1%;
+        cursor: pointer;
+    }
+
+    sendButton:hover {
+        background-color: #eeee;
+    }
+
+    .WhereToFindUsBlock{
+        background-color: #81BDBE;
+        padding-bottom: 32px;
+    }
+
+    .WhereToFindUsBlock > button {
+        margin-top: 1cm;
+    }
+
+    .WhereToFindUsBlock > h2 {
+        margin-left: 12.5%;
+        color: white;
+        padding-top: 1%;
+        font-size: 24px;
+    }
+
+    iframe {
+        margin-left: 12.5%;
+        position: relative;
+        border: 2px solid #447399;
+        width:75%;
+        height:500px
+    }
+
+    .CommentBlock > h2 {
+        position: relative;
+        font-size: 24px;
+        padding-left: 5%;
+        margin: 32px 0px
+    }
+
+    .CommentBlock > h3 {
+        position: relative;
+        padding-left: 5%;
+        margin: 32px 0px
+    }
+
+    .emailConfirmation {
+        border: 2px solid green;
+        color: green;
+        border-radius: 5px;
+        width: 250px;
+        text-align: center;
+        position: relative;
+        margin: 0px auto;
+        display: block;
+        margin-top: 1%;
+        font-size: 24px;
+    }
+
+    @media (max-width: 800px) { 
+        ul {
+            flex-direction: column;
```

Comment:
Have you checked what this looks like?? The div is bigger than the image below 800px which defo shouldn't be the case as image is the focus and the entire div is clickable not just the buttons?